tests/core/test_azure.py

```diff
@@ -54,9 +54,11 @@ class TestAzure:
     def test_get_credentials(self, mocker: MockFixture) -> None:
         mocked_default_creds = mocker.patch("opta.core.azure.DefaultAzureCredential")
         Azure.get_credentials()
+        Azure.credentials = None
         mocked_default_creds.assert_called_once_with()
 
     def test_get_remote_config(self, mocker: MockFixture, azure_layer: Mock) -> None:
+        Azure.credentials = None
```

Comment:
can we use [this](https://stackoverflow.com/a/22638709) instead of having this code in every test?